wbooks/src/services/BookService.ts

```diff
@@ -0,0 +1,12 @@
+import { BOOKS_MOCK } from '@constants/mockBooks';
+import { BookServiceMockedResponse } from '@interfaces/bookServiceMock';
+
+const BookService = {
+  getBooks: (): Promise<BookServiceMockedResponse> =>
+    new Promise(resolve => {
+      const data = { ok: true, data: BOOKS_MOCK, problem: 'Something went wrong' };
```

Comment:
If `ok` === `true`, why the `data` has a `problem`? The problem is only when `ok: false`

Reply:
Added a reject case to handle that scenario.